.env.sample

```diff
@@ -27,6 +35,8 @@ GITEA_ASSIST_AUTH_TOKEN="YOUR_BEARER_TOKEN"
 CANVAS_API_KEY="YOUR_API_KEY"
 CANVAS_API_URL="https://uncch.instructure.com/api/v1"
 CANVAS_COURSE_ID="12345"
+CANVAS_COURSE_START_DATE="1970-01-01T00:00:00Z"
```

Comment:
Currently in my .env.sample file, I have this variable set to this format for a date: `CANVAS_COURSE_START_DATE="06/26/2024 00:00:00"`. Are you sure the format you're using works? If so that's cool, just wanna make sure.

Reply:
yes it's an iso formatted datetime